operations/windows-cell.yml

```diff
@@ -51,6 +52,11 @@
               ca_cert: "((diego_bbs_client.ca))"
               client_cert: "((diego_bbs_client.certificate))"
               client_key: "((diego_bbs_client.private_key))"
+            consul:
+              ca_cert: "((consul_agent.ca))"
```

Comment:
@mhoran I think I understand.  In that case, is there some way that we can make this difference more obvious?  I think that calling it `ca_cert` (which looks so much like our other properties where the ca_cert is used to sign the certificate which is located adjacent in the manifest) makes it especially easy to misunderstand in this case.  @dsabeti && @anEXPer , do you want to weigh in here?

Thanks,
Josh && @mingxiao